site_config/docs2-0-0.js

```diff
@@ -488,6 +488,15 @@ export default {
           },
         ],
       },
+      {
+        title: 'Integration',
```

Comment:
Maybe we should use this in Chinese
```suggestion
        title: '集成',
```